File: java/MaratonaJava/javinha/src/main/java/com/maratonajava/estrutura_sequencial/exerc07.java
package com.maratonajava.estrutura_sequencial;

import java.util.Scanner;

public class exerc07 {
    public static void main(String[] args){
        /*Faça um Programa que calcule a área de um quadrado, 
        em seguida mostre o dobro desta área para o usuário*/
        try(Scanner areaQuadrado = new Scanner(System.in)){
            double valorQuadrado, calcQuadrado, dobroQuadrado;

            System.out.println("Programa Calcular o Raio de um círculo.");
            System.out.println("================================================");
            System.out.println("");

            System.out.println("Informe o valor de um Quadrado.");
            valorQuadrado = areaQuadrado.nextDouble();

            calcQuadrado = (valorQuadrado*valorQuadrado);
            dobroQuadrado = calcQuadrado*2;

            System.out.println("O valor da área de um Quadrado é:"+calcQuadrado+", e seu dobro é: "+dobroQuadrado);
        }

    }
}

File: C/Exercicios/Estrutura_Sequencial/exerc07.c
#include <stdio.h>

#define descricao_programa "Programa que calcule a área de um quadrado, em seguida mostre o dobro desta área para o usuário."

int main(){
    printf("%s.\n",descricao_programa);

    float area_quadrado, ladoQuadrado,dobroQuadrado;

    printf("Informe o valor do lado de um Quadrado.\n");
    scanf("%f",&ladoQuadrado);

    area_quadrado = ladoQuadrado*ladoQuadrado;
    dobroQuadrado = area_quadrado*2;

    printf("O valor da área do Quadrado é: %f.\n",area_quadrado);
    printf("O valor dobrado da área do Quadrado é: %f.\n",dobroQuadrado);
    
}